quick-start/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link type="image/png" rel="icon" href="https://raw.githubusercontent.com/GMartigny/pencil.js/master/media/mini-logo.png">
+    <title>Quick-start - Pencil.js</title>
+    <link type="text/css" rel="stylesheet" href="../res/reset.css"/>
+    <link type="text/css" rel="stylesheet" href="../res/style.css"/>
+</head>
+<body>
+<div class="title-screen">
+    <div class="main-title">
+        <h1>
+            Quick-start
+        </h1>
+        <small>
+            Clear step-by-step demonstration of Pencil.js capabilities
+        </small>
+    </div>
+    <div class="element description">
+        <p>
+            Come along to create a nice animation in <strong>50 lines of codes</strong> in 9 steps.<br>
+            You can easily follow as each steps introduce <strong>only one</strong> feature at a time.<br>
+            This is the <strong>easiest way</strong> to get to know Pencil.js and should take about 5mn to read.
```

Comment:
```suggestion
            This is the <strong>easiest way</strong> to get to know Pencil.js and should take about 5 minutes to read.
```
I've never seen `mn` as an abbreviation for minutes. The full word is more accessible.